### DISCUSSION TITLE
AT32F423 can swap AT32F415 ?

### DISCUSSION THREAD

**Discussion by dinosauria123** [keirf/flashfloppy#956] [Q&A]
AT32F415 is small sram (32kB).

AT32F423CCT7 (LQFP48) is seems to pin compatible to AT32F415 and is has bit larger (48kB) sram.

Is it possible to work flashfloppy  to swap AT32F423 instead of AT32F415 ?

https://www.arterychip.com/en/product/AT32F423.jsp

https://www.arterychip.com/download/MG/MG0023_Migrating_from_AT32F415_to_AT32F423_EN_V2.0.0.pdf

**Comment by keirf** [comment]
The AT32F415-based Goteks do not connect Vdd supply at LQFP48 pin 36, because that is a GPIO on AT32F415. But it is Vdd on AT32F423. So this won't work.

If you want a better chip, you should get an AT32F435-based Gotek.

**Reply by dinosauria123** [reply]
Thank you for your comment.

**Reply by dinosauria123** [reply]
I found schematics of gotek AT32F415 board.

https://sysadminmosaic.ru/_detail/gotekemulator/sfr1m44-u100k/at32f415-gotek_schema.png?id=en%3Agotekemulator%3Asfr1m44-u100k%3Aat32f415

This seems to show PF7 (pin36) is not connected.

How do you think if we get more 16kB sram ?

**Reply by keirf** [reply]
But it needs to be connected for AT32F423. I don't care a great deal for improving the 415 boards as there is already the 435 board option for those who care.

**Reply by dinosauria123** [reply]
Ok, thank you for your great work !
